.eslintrc.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   root: true,
-  extends: "babel",
+  parser: "lightscript-eslint",
```

Comment:
Okay, it looks like we should instead be adding `babel-eslint` as a devDependency so the ES6 code in the repo can be linted?